### PR TITLE
Mini challenge 1

### PR DESCRIPTION
**Pull request by laulujan**
## Description
```
< First deliverable: Mini challenge 1 >

- Created UI layout for homepage and styled with styled-components.
- Get videos information from [this mock file](https://raw.githubusercontent.com/wizelineacademy/react-gist/main/capstone-project-1/mocks/youtube-videos-mock.json)
- 
```
## Type of Change

```
Delete options that are not relevant.
```

- [x] New feature (non-breaking change which adds functionality)
- [x] Styles (Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc))
- [x] Builds (Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)

## How should this be tested?
```

- Manual exploratory testing:
- 
```
## Any background context you want to provide?
```
This is the first deliverable of the capstone 1, includes only UI layout for home page styled with Styled-components and fetching data from the mock file
```
## What is the deliverable?
```
Deliverable1: Mini-Challenge 1: Core Concepts and Styling. 
```
## Screenshots and links
Find the live version [HERE](https://quirky-dubinsky-3042f5.netlify.app/)
Screen Shot 2022-01-10 at 9 48 05

## Notes
```
Complete responsiveness still in progress

```
## Checklist
- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my deliverable works.
- [x] I reviewed existing Pull Requests before submitting mine.
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/api/mockApi.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const getVideos = () => {
+  return fetch(
+    'https://raw.githubusercontent.com/wizelineacademy/react-gist/main/capstone-project-1/mocks/youtube-videos-mock.json'
+  ).then((response) => response.json());
+};
+// export const getVideos = async (url) => {
```
Comment (ragrum) [inline src/api/mockApi.js]: Remember, we have Git, so it's a good practice not to include commented out code.

File: src/components/LoginMenu/LoginMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import { LoginMenuButton, Avatar } from './LoginMenu.styled';
+import Dropdown from '../Dropdown/Dropdown';
+
+const LoginMenu = () => {
+  const [isDropDownOpen, setIsDropDownOpen] = useState(false);
+
+  const handleClick = () => {
+    setIsDropDownOpen(!isDropDownOpen);
+    console.log('open dropdown');
```
Comment (ragrum) [inline src/components/LoginMenu/LoginMenu.jsx]: Remove `console` statements.

File: src/components/Menu/Menu.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import { Button, MenuIcon } from './Menu.styled.js';
+import Dropdown from '../Dropdown/Dropdown';
+
+const Menu = () => {
+  const [isDropDownOpen, setIsDropDownOpen] = useState(false);
+
+  const handleClick = () => {
+    setIsDropDownOpen(!isDropDownOpen);
+    console.log('open dropdown');
```
Comment (ragrum) [inline src/components/Menu/Menu.jsx]: Dito.

File: src/pages/Home/Home.page.jsx [hunk 4]
```diff
@@ -1,38 +1,23 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React, { useState, useEffect } from 'react';
+//import { Link, useHistory } from 'react-router-dom';
```
Comment (ragrum) [inline src/pages/Home/Home.page.jsx]: Remove commented out code, also the one below.

**Review by ragrum — APPROVED** [review]
Left some minor comments, please address and wait to complete the subsequent stacked PRs before merging.

File: src/api/mockApi.js (hunk 1 above)
Comment (laulujan) [reply src/api/mockApi.js]: Commented out code and console log statements were corrected on the following [PR ](https://github.com/laulujan/react-apprenticeship-capstone1/pull/2)

**Pull request merged**